ENIGMAsystem/SHELL/Graphics_Systems/Direct3D11/DX11vertex.cpp

```diff
@@ -16,23 +16,279 @@
 **/
 
 #include "Graphics_Systems/General/GSvertex_impl.h"
+#include "Graphics_Systems/General/GScolor_macros.h"
+
+#include "Bridges/General/DX11Context.h"
+
+#include <D3Dcompiler.h>
+
+namespace {
+const char* g_strVS =
+  "struct VertexInputType {\n"
```

Comment:
It's 2018; use `R"()"`.

Reply:
Ok, done: 80f8e73c7cb5f371c5e19260cfe4029b350358cf